### DISCUSSION TITLE
Two easy features for signing PDFs

### DISCUSSION THREAD

**Discussion by Softbox44** [Stirling-Tools/Stirling-PDF#1083] [Ideas]
Regarding signing PDFs: It would be great to have the following option:
- would be great to have an option to send out PDFs to the other person after signing. So setup screen for an IMAP server incl sender email and a One-liner to add the recipient email would be great. Is that possible?
- would be great to be able to move a signature to several pages once its uploaded into SterlingPDF.

**Comment by Softbox44** [comment]
ALso being able to adding a date field and name (text) when signing would be great.

**Comment by smartsammler** [comment]
I would also highly appreciate to move the certificate signature on the page like it is possible for the hand-written signature (or in Acrobat). This is useful if one has got a PDF with a checklist to sign the progress or if multiple users have to sign a form.

**Comment by AndyBodnar** [comment]
Great feature suggestions for PDF signing! Here are some thoughts:

**1. Signature positioning improvements:**
- Drag-and-drop placement would be huge for UX
- Preview overlay showing exactly where signature will appear
- Save favorite positions per document type (invoices always bottom-right, etc.)

**2. Signature library/management:**
- Save multiple signatures (formal, initials, dated stamp)
- Quick-select from recent signatures
- Import from image files

**Implementation considerations:**
- PDF.js or similar for the preview/positioning UI
- Store signatures as base64 in local storage or user profile
- Could leverage existing image-to-PDF functionality

**Existing workarounds:**
- Use the stamp feature with a pre-made signature image
- Add signature via the image overlay tool

These seem like reasonable enhancements that would improve the signing workflow significantly. The positioning especially - current signature placement in most PDF tools is trial and error.

+1 for both features!
